### ISSUE TITLE
Missing French translation

### ISSUE THREAD

**Issue by FraBoCH** [jtc42/monopong-ios-issues#6]
Thanks for this awesome game ! I see there is no french translation, and I would like to help add it. Just send me the files to translate 😉

**Comment by jtc42** [comment]
Absolutely 5 star coincidence I got the French localisation back earlier today. What I might do is post the draft here for you to cross check? Of course you'll go in the credits if you're happy with that arrangement?

**Comment by FraBoCH** [comment]
That’s funny. I’ll be happy to cross-check. We may need to be careful with the length of labels as French names are often longer that English ones.

**Comment by jtc42** [comment]
The current draft translation is:

```
"start";  DÉMARRER
"pause"; PAUSE
"resume"; REPRENDRE
"quit"; QUITTER
"again"; // Play again/restart - RECOMMENCER
"back"; RETOUR

"credits"; CREDITS
"help"; // Help/how to play - AIDE

"tap to play"; CLICK POUR JOUER
"style skip"; // Skip to a  specific style/level - PASSER - CHANGER NIVEAU

"perfect"; // Hit the ball perfectly (dead-center of paddle) - PARFAIT
"corner"; // Hit the ball with the very corner of the paddle - CORNER
"recovery"; // As in, you messed something up and then recovered/saved it - RECUPERATION
```

**Comment by FraBoCH** [comment]
Here is my translation, keeping the same casing you have in the game and with the proper accents: 
"start"=« démarrer »
"pause"= « pause »
"resume"=« reprendre »
"quit"=« quitter »
"again"=« encore »
"back"=« retour »

"credits"=« crédits »
"help"=« aide »

"tap to play"=« toucher pour jouer »
"style skip"=« passer » // not sure about this one. Can you provide a screenshot maybe ?

"perfect"=« parfait »
"corner"=« bord » 
"recovery"=« récupération »
I saw in the game that you combine those two labels in some cases. However in French, the order should not be the same to be correct. So I would say « corner recovery »=« récupération au bord »

**Comment by jtc42** [comment]
Hi @FraBoCH! Sorry for the delay getting this implemented. I've added basic localisation now, but as you pointed out things like combining labels won't work perfectly yet. 

Doing that properly will be on the todo list, but I'll need to have separate logic for each language, or hard-code each combination possible. I've not decided which yet, and have been super busy moving house recently so right now I'd rather get _something_ released and perfect it down the line.

Would you like to be listed in the translator credits, and if so, how? I can just use "github.com/FraBoCH" or "FraBoCH" if you like.

**Comment by FraBoCH** [comment]
Hi @jtc42! No problem. I agree with you, an imperfect localization is better than nothing. No need for credits as far as I’m concerned. I’m happy to help a particular particularly well designed and implemented game, just for fun 🤩

**Comment by jtc42** [comment]
Thank you, really appreciate that!

**Comment by jtc42** [comment]
Closing this as it's resolved by the 1.3 release. Submission likely today, release some time in the next week or so (depending on Apple review etc)
